File: QB_Challenges/PrepInsta/01_Getting_Started/01_Negative_Or_Positive/Java/NOP_Nested_IfElse.java
import java.util.Scanner;

public class NOP_Nested_IfElse {
    

    public static void NOP() {
    Scanner sc = new Scanner(System.in);
    System.out.print("Enter the Number: ");
    int num = sc.nextInt();

    if(num !=0){
        if(num>0){
        System.out.println(num+" is positive Number");
    }else{
            System.out.println(num+" is negative Number");
        }
    }else{
        System.out.println(num+" is zero");
    }
    }

    public static void main(String[] args) {
        NOP();
    }
}

File: QB_Challenges/PrepInsta/01_Getting_Started/01_Negative_Or_Positive/CPP/NOP_Nested_IfElse.cpp
#include <iostream>
using namespace std;

void NOP(){
    int num;
    cout <<"Enter the number: ";
    cin >>num;

    if(num != 0){
        if(num>0){
            cout <<num <<" is positive number";
        }else{
            cout <<num <<" is negative number";
        }
    }else{
            cout <<num <<" is zero number";
    }
}

int main(){
NOP();
  return 0;
}